mlir/lib/Dialect/NVGPU/Transforms/Utils.cpp

```diff
@@ -15,46 +15,6 @@
 using namespace mlir;
 using namespace mlir::nvgpu;
 
-Operation::operand_range nvgpu::getIndices(Operation *op) {
```

Comment:
... this could be an interface or something, idk

Or an annotation on arguments

Or perhaps refactored out to GPU Common Indexing Utils or something